conf/webpack-devserver.conf.js

```diff
@@ -31,6 +34,11 @@ module.exports = {
                 loaders: ['babel-loader']
             },
             {
+                test:    /\.js$/,
+                include: [resolve(__dirname, "../node_modules/panda-session")],
```

Comment:
Unfortunately not... without clever regex-foo, simply adding include will cause everything else to be excluded, and I don't really want to have to put the public path in the include... also like the separation of a separate rule, can remove once uglify get it's act together